prow/gerrit/adapter/adapter.go

```diff
@@ -376,7 +381,16 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 	case client.Merged:
 		postsubmits, err := cache.GetPostsubmits(trimmedHostPath, func() (string, error) { return baseSHA, nil }, func() (string, error) { return change.CurrentRevision, nil })
 		if err != nil {
-			return fmt.Errorf("failed to get inRepoConfig for Postsubmits: %w", err)
+			attempts := 1
```

Comment:
seems like this loop will only run once if attempts starts at 1 and max is 2

Reply:
Yea. Originally I had it do 2 retries, but lowered it to one. I kept the loop because I would like us to be able to change the inRepoConfigRetries var without worrying about the loop code. If you disagree I can get rid of the loop and the var.

Reply:
ah. I see, I misread the code. How about:

```
var postsubmits []Postsubmit
for attempt := 0; attempt < inRepoConfigRetries; attempt++ {
  postsubmits, err = cache.GetPostsubmits(trimmedHostPath, func() (string, error) { return baseSHA, nil }, func() (string, error) { return change.CurrentRevision, nil })
  if err == nil {
    break
  }
}
if err != nil {
    return fmt.Errorf("failed to get inRepoConfig for Postsubmits: %w", err)
}
```

Reply:
Typically I try to avoid break statements because I find them hard to read, but I think you're right, that is much cleaner.